config/application.rb

```diff
@@ -22,5 +22,6 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.